js-sdk/providing_static_data.mdx

```diff
@@ -122,30 +122,57 @@ If you use namespaces, it should look like this:
 
 ## Data fallbacks
 
-You can provide local data as a fallback for CDN, for cases when network request fails.
+You can provide local data as a fallback for CDN, for cases when a network request fails.
 
 ```ts
 const tolgee = Tolgee()
-  .use(BackendFetch({
+ .use(BackendFetch({
     prefix: '<your content storage link>',
     fallbackOnFail: true
-  }))
-  .init({
+ }))
+ .init({
     staticData: {
       en: () => import('./i18n/en.json'),
       de: () => import('./i18n/de.json'),
-    }
-    ...
-  })
+ }
+ ...
+ })
 ```
 
-`fallbackOnFail` option will tell Tolgee to look for data in the next source, so it will fetch your local data. **Order of plugins is important here, they are used in the same order as they were added.**
+`fallbackOnFail` option will tell Tolgee to look for data in the next source so it will fetch your local data. **Order of plugins is important here, they are used in the same order as they were added.**
 
-:::info Plugins are used in the same order as they were added.
-Tolgee first checks if data aren't already in cache (data passed directly to `staticData`) then all the plugins and then dynamic data from `staticData` field.
+### Fallback order
+
+Tolgee tries to find the data in following way:
+
+ 1. If available, uses plain object from `staticData`
+ 2. Tries to fetch language data with plugins (in the same order as they were added)
+ 3. Fetches language data through `staticData` dynamic properties
```

Comment:
```suggestion
 3. Fetches language data through `staticData` dynamic properties returning function
```

I am not sure I would understand. Isn't this more clear?

Reply:
Ok, fixed